recipe/meta.yaml

```
@@ -1,36 +1,33 @@
 {% set name = "spatialpandas" %}
-{% set version = "0.4.10" %}
+{% set version = "0.5.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 032e24ebb40f75c5c79cb79d7c281f2990e69ba382c0b24acb53da7bba60851c
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 412bb93bb25f49b6c057e825560d14c06e71137228fa99dc007b3a191bfbabd1
 
 build:
   number: 0
-  skip: True  #[py<39 or s390x]
+  skip: True  #[py<310 or s390x]
   # skipping s390x as it is not available for numba or pyarrow
   script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
-    - pip
     - python
-    - param
-    - wheel
-    - pyct
-    - setuptools
+    - pip
+    - hatchling
+    - hatch-vcs
   run:
-    - dask-core
-    - fsspec
-    - numba
-    - pandas
-    - param
-    - pyarrow >=1.0
     - python
+    - dask-core >=2025.1
+    - fsspec >=2022.8
+    - numba
```

```suggestion
    - numba
    - packaging
```
https://github.com/holoviz/spatialpandas/blob/v0.5.0/pyproject.toml#L33